### PR TITLE
Added Dutch translation

### PR DESCRIPTION
**Pull request by Vistaus**
🙂

### CODE REVIEW

File: public/_locales/nl/messages.json [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+        "message": "Logboek"
+    },
+    "donate": {
+        "message": "Doneren"
+    },
+    "help": {
+        "message": "Hulp nodig?"
+    },
+    "about": {
+        "message": "Over"    },
```
Comment (AXeL-dev) [inline public/_locales/nl/messages.json:51]: Can you fix the closing bracket position here please? 😅

File: public/_locales/nl/messages.json [hunk 2]
```diff
@@ -0,0 +1,298 @@
+{
+    "appName": {
+        "message": "Niet storen s.v.p."
```
Comment (AXeL-dev) [inline public/_locales/nl/messages.json:3]: The app name translation looks a bit weird to me :thinking:, i tried to translate it online & got `Leid me niet af` [on google translate](https://translate.google.com/?sl=en&tl=nl&text=don%27t%20distract%20me&op=translate), i'm not a native speaker so i don't know which one is really more relevant, WDYT?

File: public/_locales/nl/messages.json (hunk 1 above)
Comment (Vistaus) [reply public/_locales/nl/messages.json:51]: Oops, fixed!

File: public/_locales/nl/messages.json (hunk 2 above)
Comment (Vistaus) [reply public/_locales/nl/messages.json:3]: There are many possible translations, but ‘Niet storen s.v.p.’ is a common phrase to indicate that someone doesn't want to be disturbed or distracted. But ‘Geen afleiding’ would be a good translation as well. Do you want me to change it to that?

Comment (AXeL-dev) [reply public/_locales/nl/messages.json:3]: Finally, that's okay let's go for ‘Niet storen s.v.p.’

Thanks for the updates btw!

**Comment by Vistaus** [comment]
No problem, and thanks for merging! 🙂

**Pull request merged**